src/app/core/components/account-settings/account-settings.component.ts

```diff
@@ -98,9 +100,32 @@ export class AccountSettingsComponent implements OnInit {
 
   async onChangePasswordFormSubmit(value: AccountPasswordVOData) {
     this.waiting = true;
+    let trustToken = null;
 
     try {
-      await this.api.auth.updatePassword(this.account, value);
+      try {
+        const loginResp = await this.accountService.checkForMFAWithLogin(
+          value.passwordOld,
+        );
```

Comment:
I expect if the user alters this response to claim that they don't need MFA then the call will just fail below since they won't have a trust token. Am I understanding correctly?